src/Raven.Client/Documents/Operations/DatabaseHealthCheckOperation.cs

```diff
@@ -14,11 +15,16 @@ public RavenCommand GetCommand(DocumentConventions conventions, JsonOperationCon
 
         private class DatabaseHealthCheckCommand : RavenCommand
         {
+            public DatabaseHealthCheckCommand()
+            {
+                Timeout = TimeSpan.FromSeconds(15); // maybe even less?
```

Comment:
Shouldn't we have a single location to define the default timeout?

Reply:
No, that would affect too many things. All internal'ish commands have this timeout set manually, In theory we could introduce sth like RavenInternalCommand which would set it by default.